Scripts/CI/README_Metadata_StyleCheck/entry.py

```diff
@@ -0,0 +1,564 @@
+import os
+import argparse
+import subprocess as sp
+from os import listdir
+from os.path import isfile, join
+import json
+
+def main():
+    # Tell the program that we're using global variables
```

Comment:
suggestion - add an "example usage" comment here:
e.g.:

'''
  Example Usage:
  python3 entry -s /path/to/samples/ArcGISRuntimeSDKQt_CppSamples/Maps/DisplayMap
'''